Nodes/Screen_Node/Screen_Node.ino

```diff
@@ -18,15 +22,18 @@ union MsgData
 millisDelay CAN_Send_Delay; 
 void setup()
 {
+  /*******************************************************************************/
   pinMode(PIN_CAN_STANDBY, OUTPUT);
   digitalWrite(PIN_CAN_STANDBY, false); // turn off STANDBY
   pinMode(PIN_CAN_BOOSTEN, OUTPUT);
   digitalWrite(PIN_CAN_BOOSTEN, true); // turn on booster
 
 
   Serial.begin(115200);
-  // start the CAN bus at 500 kbps
+  // start the CAN bus at 500 kbps (kilo-bits/second)
     CAN.begin(500000);
+
+  CAN_Send_Delay.start(10); //10 ms -> 100 Hz (Receiving 2 messages at 100 Hz, so 200 messages/sec)
```

Comment:
You don't necessarily need to match the delays to the sender here if you don't want.  You can display it less often or you can have it print to serial out on receipt. It may or may not keep up as i know Serial print is slow. 

All good.